docs/content/latest/quick-start/build-apps/java/ysql-spring-data.md

```diff
@@ -286,7 +286,7 @@ $ curl http://localhost:8080/orders
         "userId": 2,
```

Comment:
Just FYI, the format of these code blocks could be `output.json` instead of `json`, and that would remove the copy buttons. Not worth doing in this PR, as there are many more scattered all over the place. Just a reminder the option exists :)

Reply:
Thank you! Sure, will make a note of this for future ones.